Assets/MixedRealityToolkit/Inspectors/MixedRealityPlayspaceInspector.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    [CustomEditor(typeof(MixedRealityPlayspace))]
+    public class MixedRealityPlayspaceInspector : UnityEditor.Editor
+    {
+        private static string description =
+          "This is where the Toolkit instantiates objects like cursors, pointers and boundary visualizations." +
+          "\n- Your main camera should be kept here (unless you are using a custom CameraSystem implementation.)" +
+          "\n- This transform can be moved, but it should not be parented under another object." +
```

Comment:
>but it should not be parented under another object [](start = 44, length = 50)

Why this advice that the playspace shouldn't be parented under another object? Does that actually break anything? An additional transform in the scene graph is an important capability for the app, since they can't directly manipulate the camera's transform (it will be overwritten), nor the playspace transform (it's already in use for teleport etc.).

Reply:
I'm probably being over-cautious. I'll remove that part of the warning message. I've tested parented playspace transforms and they do work.